app/services/MethodOfPaymentService.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.ReferenceDataConnector
+import models.SelectableList
+import models.reference.{AdditionalInformation, MethodOfPayment}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class MethodOfPaymentService @Inject() (referenceDataConnector: ReferenceDataConnector)(implicit ec: ExecutionContext) {
+
+  def getMethodOfPaymentTypes()(implicit hc: HeaderCarrier): Future[Seq[MethodOfPayment]] =
```

Comment:
Do we need to sort these in any particular order?

Reply:
we need Other option at the end. so we won't be sorting this in any order.

Reply:
I guess in Production we have little control over the order that the data gets sent to us. But fair enough if the order in the content doc does not follow any logical order from the code and/or description values then happy to leave unsorted. Not as big a deal for radio buttons.